docs/docs/deploying-and-hosting.md

```diff
@@ -8,3 +8,5 @@ Getting your shiny new Gatsby site onto the cloud is probably the first thing yo
 In this section you'll find answers to questions about dealing with app routes that aren't at the root of your domain with `path prefixes`, as well as how to deploy to countless hosting solutions such as [Netlify](https://www.netlify.com/), [AWS Amplify Console](https://console.amplify.aws), and [GitHub Pages](https://pages.github.com/). As well as what's necessary to make sure you're fully prepared for a deploy! 🚀
 
 [[guidelist]]
+
+**Note:** Gatsby deployment is not limited to the above mentioned list and you can deploy anywhere you wish to as long as you setup appropriate [caching rules](https://www.gatsbyjs.org/docs/caching/) for best performance of your site.
```

Comment:
Can we make this clearer by putting this up top? I think it's important enough to be explicit here!

Reply:
Mentioning that the list down below is incomplete without the list in context seems a bit odd to me I guess

Reply:
Right - so you wouldn't move this _exact_ content up, you'd move a similar disclaimer up to the top! i.e. something like (feel free to re-word this, wrote this quickly)

```
The result of Gatsby's build process is static content: HTML, CSS, JavaScript, images, and more that power your great web application. This content that is produced by this build process is incredibly easy and affordable to host with any provider. We have several guides below for hosting with particular providers, but this list is by no means exhaustive. Whether you're deploying with AWS, Netlify, or something in between--switching to static content makes deploying a trivial concern.
```